Eint 33 update readme

sahil-rokt opened the pull request:
### Background ###

Updated the Readme.md for ReactNative sdk.

Fixes [(issue)]

(https://rokt.atlassian.net/browse/EINT-33)

### What Has Changed: ###

List out what has changed as a result of this PR.

### How Has This Been Tested? ###

Describe the tests that you ran to verify your changes. 

### Notes

Add any notes or extra information here that might be useful to the reviewer if applicable i.e. links to documentation/blogs or dashboards.

### Checklist: ###

- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have verified that CI completes successfully.
- [ ] All insignificant commits have been squashed.

DJingSeo reviewed (commented):
Looks good to me but I can see some security issues raised by a dependantbot. I was wondering what's your plan to address it?

sahil-rokt commented:
> Looks good to me but I can see some security issues raised by a dependantbot. I was wondering what's your plan to address it?

I have updated the dependency and added package.lock
